integration-tests/neo4j/src/main/java/io/quarkus/it/neo4j/Neo4jResource.java

```diff
@@ -73,7 +73,7 @@ public String doStuffWithNeo4j() {
     @Path("/reactive")
     @Produces(SERVER_SENT_EVENTS)
     public Publisher<Integer> doStuffWithNeo4jReactive() {
-
+driver.rx
```

Comment:
I'm pretty sure you didn't want that in your commit :).

Could you fix and amend your commit?